mitmproxy/http.py

```diff
@@ -140,6 +140,7 @@ class HTTPFlow(flow.Flow):
     """
     request: HTTPRequest
     response: Optional[HTTPResponse] = None
+    trailers: http.Headers
```

Comment:
I suppose trailers are optional as well?

Reply:
fixed it